code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -241,6 +252,16 @@
 	new /obj/item/clothing/suit/storage/vest/merc(src)
 	new /obj/item/clothing/head/helmet/merc(src)
 
+/obj/item/weapon/storage/box/syndie_kit/mafia
+	name = "mafia kit"
+	desc = "Put heads in this box and ship it off to their loved ones after you're done taking the contents out."
+
+/obj/item/weapon/storage/box/syndie_kit/mafia/fill()
+	..()
+	new /obj/item/clothing/under/gentlesuit/traitor(src)
+	new /obj/item/clothing/head/fedora/sharp(src)
+	new /obj/item/clothing/shoes/leather(src)
+
 /obj/item/weapon/storage/secure/briefcase/money
 	name = "suspicious briefcase"
```

Comment:
I´d suggest changing the names away from "suspicious briefcase" to avoid security out right validing the antag because the "briefcase looks suspicious".

Same with the "bad feeling" in the description

Reply:
Sure. I'll include it ithis update.